scripts/binaryLayer/binaryLayer.yml

```diff
@@ -20,5 +20,5 @@ outputs:
   output_binary: 
     label: binary layer output
     description: land cover raster classes, where 1 is above threshold and 0 below threshold.
-    type: image/tiff;application=geotiff[]
-    example: y2000_class210_binary.tif
\ No newline at end of file
+    type: image/tiff;application=geotiff
```

Comment:
This is inconsistent with the script's code. It can output an array if we provide many classes as inputs. That would cause some crashes.

If we believe it should instead output a single binary that merges all the classes given in input (what I actually thought it would do, at first), then we should modify the code to do so...

Reply:
I think this was a change @gottacatchenall  made, but my guess is that AssignID didn't like it when it was an array. This is the same problem we have with other of Juan's scripts that output multiple layers.

Reply:
Yeah I did, I'm not really sure what the AssignID block is for, is it not necessary if the goal is only to propagate the path of the layer into the next script?

Reply:
Ya we can actually get rid of it in that arm of the pipeline it looks like. It's a way to selectively run an arm of the pipeline if you request it or not, but I think we'll be doing the mask every time.

Reply:
So that's settled, we can remove this AssignId and directly link the tiff to the mask input.

About the R code that produces the mask, should we remove the loop, make a merged mask of the selected layers?

Reply:
I think merging the set of inputs into a single mask is the most useful and requires the least disk storage